File: OOP/LY_THUYET/ClassExam/Session 2/Ex4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
// Nhập mảng , tính tổng mảng
namespace Ex4
{
    public class Ex4
    {
        static void Main(string[] args)
        {
            Console.Write("Nhap so phan tu cua mang : ");
            int n = Convert.ToInt32(Console.ReadLine());    
            int[]arr= new int[n];
            int sum = 0;
            for(int i = 0; i < n; i++)
            {
                Console.Write("Nhap gia tri thu {0} : ", i);
                arr[i] = Convert.ToInt32(Console.ReadLine());
                sum += arr[i];
            }
            Console.WriteLine("Tong cac phan tu trong mang : " + sum);
        }
    }
}

File: TriTueNhanTao/Lesson1/SampleExercise/Ex4.py
#Viết chương trình tính tổng các số từ 0 đến n mà là bội của 3 hoặc 5
sum = 0
n = 11
for i in range(n):
    if i % 3 == 0 or i % 5 == 0:
        sum += i

print(f"Tổng các số từ 0 tới {n} mà là bội của 3 hoặc 5 là : {sum}")
        